.bazelversion

```diff
@@ -1,4 +1,4 @@
-5.1.0
+5.2.0
```

Comment:
Why not 5.3?

Reply:
5.3.0 breaks e2e/bzlmod in a strange way